REPORT.md

```diff
@@ -0,0 +1,76 @@
+# Reporting capabilities within zAppBuild
+
+zAppBuild provides a set of reporting capabilities, which are part of the build framework itself to address some common demands of mainframe development teams.
+
+Developers are particularly interested if
+* a change of an element impacts other application components which are managed in a different repository.
+* their changes on an isolated feature branch potentially cause a conflict with some concurrent development done by others within the same repository.
+
+While during the analysis phase of a new task, the developer leverages application dependency understanding tools such as IBM Application Discovery, it is beneficial to enable also the build framework to automatically generate reports to answer the above questions.
+
+By default these reporting capabilities are turned off, while they don't belong to the core functionalities of a build framewoek.
+
+## Report External Impacts 
+
+### Purpose
+
+The _Reporting External Impacts_ feature enables the build framework to generate reports about impacted files in other application contexts to document cross application impacts for files that were changed within your application scope.
```

Comment:
Just a streaming thought - So this says, "I made a change in my application.  Tell me who is impacted in the other applications".  What about the other way around?  Will I be able to tell if a change in the other application impacts my application?  Or, is the analysis function provided in the framework is from a application point of view outward into the other applications.